packages/@o3r/testing/package.json

```diff
@@ -101,6 +105,7 @@
     "pngjs": "^4.0.1",
     "protractor": "^7.0.0",
     "rxjs": "^7.4.0",
+    "temporal-polyfill": "^0.1.1",
```

Comment:
Should be optional as required only by the Kassette sub entry

Reply:
done